src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -92,9 +94,212 @@ def process_declarations(code):
 '''
 
 
+# Function check_bound
```

Comment:
You don't need this comment - it's in the docstring.

Reply:
Removed.